pandas/tests/copy_view/test_replace.py

```diff
@@ -395,6 +396,17 @@ def test_replace_chained_assignment(using_copy_on_write):
         with tm.raises_chained_assignment_error():
             df[["a"]].replace(1, 100, inplace=True)
         tm.assert_frame_equal(df, df_orig)
+    else:
+        with tm.assert_produces_warning(FutureWarning, match="inplace method"):
+            with option_context("mode.chained_assignment", None):
```

Comment:
Does it otherwise also raise a SettingWithCopyWarning?

Reply:
yes, [["a"]] currently copies